documentation/modules/exploit/windows/local/ms16_014_wmi_recv_notif.md

```diff
@@ -0,0 +1,36 @@
+## Intro
+
+This module exploits a uninitialized stack variable vulnerability present in the WMI subsystem  of `ntoskrnl`.  The vulnerability is described by MS16-014 and CVE-2016-0040.  The module allows the user to elevate privileges (NT AUTHORITY\SYSTEM)
+
+## Usage
+
+You'll first need to obtain a session on the target system.  Next, once the module is loaded, one simply needs to set the `payload` and `session` options.  From here, running the module will result in the payload being executed with system level privileges.
+
+An example session follows:
```

Comment:
Change this to be a 3rd level heading with the OS name, version and architecture (`### Windows 7 SP? x64`)